src/main/java/io/specto/hoverfly/junit/rule/HoverflyRule.java

```diff
@@ -304,6 +310,18 @@ public HoverflyRule printSimulationData() {
         return this;
     }
 
+    public void verify(RequestMatcherBuilder requestMatcher) {
+        hoverfly.verify(requestMatcher, times(1));
+    }
+
+    public void verify(RequestMatcherBuilder requestMatcher, VerificationCriteria criteria) {
+        hoverfly.verify(requestMatcher, criteria);
```

Comment:
I'd also test verify on Hoverfly directly, as people either work with that from the top, or the rule